### ISSUE TITLE
2021 week 26: rebase off upstream

### ISSUE THREAD

**Issue by adpaco-aws** [model-checking/rmc#247]

**Comment by adpaco-aws** [comment]
While rebasing we get a conflict with one of the issue templates: `.github/ISSUE_TEMPLATE/library_tracking_issue.md`
Action taken: Remove that file.

Then we get another conflict in `src/tools/compiletest/src/header.rs`, where a call to `iter_header` uses different arguments  (we edited this function in https://github.com/model-checking/rmc/pull/168). Action taken: Replace old call with new one.

**Comment by adpaco-aws** [comment]
The previous action did not work. It seems like [this commit](https://github.com/rust-lang/rust/pull/86272) has changed the way in which tests are marked as "ignored" (this was done in the `from_reader` impl. of `EarlyProps`) so I will have to figure out the best way to do this for RMC.

**Comment by adpaco-aws** [comment]
Tried again.

This time, I have accepted the changes in the `from_reader` impl. of `EarlyProps`. This removed [a change](https://github.com/model-checking/rmc/pull/168/files#diff-bf53d26b9ca67283fc825e902a64780b463c153d1b1df0c8e782da0300f08956R54-R60) we made in #168 that marked tests as "to-ignore" based on their name. I added an [equivalent change](https://github.com/model-checking/rmc/commit/fbdee664b572843147e1efe0142339c47af9400b#diff-bf53d26b9ca67283fc825e902a64780b463c153d1b1df0c8e782da0300f08956R879-R885) in that same commit. Then continued the rebase without any more problems.
